src/ExEss/Cms/AwareTrait/EntityManagerAwareTrait.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
```

Comment:
vendor/doctrine/persistence/lib/Doctrine/Persistence/ObjectManagerAware.php

Reply:
open it, I don't think it's very usable in our case ;-)